spec/namespaces/replication.yaml

```diff
@@ -0,0 +1,322 @@
+      style: simple
+    replication.stop::path.index:
+      in: path
+      name: index
+      description: Name of the data stream, index, or index alias to perform bulk actions on.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
+      style: simple
+    replication.pause::path.index:
+      in: path
+      name: index
+      description: Name of the data stream, index, or index alias to perform bulk actions on.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
+      style: simple
+    replication.resume::path.index:
+      in: path
+      name: index
+      description: Name of the data stream, index, or index alias to perform bulk actions on.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
+      style: simple
+    replication.status::path.index:
+      in: path
+      name: index
+      description: Name of the data stream, index, or index alias to perform bulk actions on.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
+      style: simple
+    replication.update_settings::path.index:
+      in: path
+      name: index
+      description: Name of the data stream, index, or index alias to perform bulk actions on.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
+      style: simple
+  requestBodies:
+    replication.start:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/replication._common.yaml#/components/schemas/Replication'
+      required: true
+    replication.stop:
+      content:
+        application/json:
+          schema:
+            type: object
+      required: true
+    replication.pause:
+      content:
+        application/json:
+          schema:
+            type: object
+      required: true
+    replication.resume:
+      content:
+        application/json:
+          schema:
+            type: object
+      required: true
+    replication.update_settings:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/replication._common.yaml#/components/schemas/UpdateSettings'
+      required: true
+    replication.create_replication_rule:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/replication._common.yaml#/components/schemas/CreateReplicationRule'
+      required: true
+    replication.delete_replication_rule:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/replication._common.yaml#/components/schemas/DeleteReplicationRule'
+      required: true
+  responses:
+    replication.start@200:
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              acknowledged:
+                type: boolean
```

Comment:
Use a `$ref` to `AcknowledgedResponseBase`,  https://github.com/opensearch-project/opensearch-api-specification/blob/bf356019ed3450d97fb4f9f68be4db86d1ed2195/spec/schemas/_common.yaml#L14